src/components/SpaceSidebarFooter.vue

```diff
@@ -1,40 +1,30 @@
 <script setup lang="ts">
-import { computed } from 'vue';
+const props = defineProps<{ space?: Record<string, any> }>();
 
-const props = defineProps<{ space: Record<string, any> }>();
 const socials = [
   {
     social: 'twitter',
     icon: 'twitter',
     path:
-      (props.space.twitter && `https://twitter.com/${props.space.twitter}`) ||
+      (props.space?.twitter && `https://twitter.com/${props.space?.twitter}`) ||
       ''
   },
   {
     icon: 'github',
     social: 'github',
     path:
-      (props.space.github && `https://github.com/${props.space.github}`) || ''
+      (props.space?.github && `https://github.com/${props.space?.github}`) || ''
   },
   {
     icon: 'earth',
     social: 'website',
-    path: (props.space.website && props.space.website) || ''
+    path: (props.space?.website && props.space?.website) || ''
```

Comment:
This line looks suspicious. :D

```suggestion
    path: props.space?.website || ''
```